pandas/core/dtypes/cast.py

```diff
@@ -1566,7 +1606,9 @@ def find_common_type(types: List[DtypeObj]) -> DtypeObj:
     return np.find_common_type(types, [])
 
 
-def cast_scalar_to_array(shape, value, dtype: Optional[DtypeObj] = None) -> np.ndarray:
+def cast_scalar_to_array(
+    shape: Tuple, value: Scalar, dtype: Optional[DtypeObj] = None
```

Comment:
we just pass shape onto np.empty, https://numpy.org/doc/1.19/reference/generated/numpy.empty.html

so could be int or tuple of int

I recall a discussion about adding an alias to pandas._typing or

```suggestion
    shape: Union[int, Tuple[int, ...]], value: Scalar, dtype: Optional[DtypeObj] = None
```

and the docstring would need to be updated too.

Reply:
Pushed the suggestion.

Adding an alias to _typing sounds like a good idea. Do we already have a PR on that or shall I submit one?

Reply:
> Do we already have a PR on that or shall I submit one?

I don't think so, go ahead.